Sources/StreamVideo/WebRTC/v2/StateMachine/Stages/WebRTCCoordinator+Joining.swift

```diff
@@ -334,6 +341,66 @@ extension WebRTCCoordinator.StateMachine.Stage {
             try Task.checkCancellation()
 
             try await coordinator.stateAdapter.restoreScreenSharing()
+
+            try Task.checkCancellation()
+
+            await reportTelemetry()
```

Comment:
Do we need to await for this? Maybe just create a task and forget about it?

Reply:
The only reason i didn't do it is because I don't prefer spinning up orphan Tasks and it didn't make much sense to me to add a task reference to the context. On the other hand, the case here is quite simple and things aren't expected to take long in any case. Updated!